### ISSUE TITLE
New Config File Format

### ISSUE THREAD

**Issue by geoffreywiseman** [geoffreywiseman/tunnel-boring-machine#38]
Although this will also solve #26, this has other effects, and deserves its own description. Basically, when looking over some example configuration files, I saw a lot of host repetition, and put some more thought into what this configuration file should look like for my current needs.

This format might be worse for someone else's needs, but at the moment, my own needs and those of my coworkers are the most important, because we're the most likely users.

```
user@ssh-host:
    name: port
    name: local-port:remote-port
    name: local-port:remote-host:remote-port
    name:
        alias: alias
        forward: port
    name: [port,port]
    name:
        alias: [alias1, alias2]
        remote-host: [port,port,port]
```

This gives the flavour of the config changes without showing all of the available options.

Basically, putting the host at the top level with an optional username and reducing the overall amount of config for my common cases.
